### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by b-3-n**
- Align start of learning rate decay with documentation.
- Use correct configuration for creation of test inputs.

### CODE REVIEW

**Comment by b-3-n** [comment]
I signed it!

**Review by benoitsteiner — APPROVED** [review]

**Comment by caisq** [comment]
@tensorflow-jenkins test this please.

**Comment by zheng-xq** [comment]
@tensorflow-jenkins test this please
